modules/ROOT/pages/ps-vpn-about.adoc

```diff
@@ -0,0 +1,203 @@
+
+[[vpn-routing]]
+== Anypoint VPN Routing
+
+Anypoint VPN supports dynamic or static routing for VPN connections.
+
+Dynamic routing::
+Your device uses Border Gateway Protocol (BGP) to advertise routes to Anypoint VPN. Use BGP routing if your device supports this protocol.
+Static routing::
+Requires you to specify the routes (subnets) in your network that are accessible through Anypoint VPN.
++
+You can remove routes for static VPN connections. 
+
+[NOTE]
+A maximum of 95 route table entries is permitted per private space, regardless of the number of VPN connections.
+Consolidate networks to the fewest number possible to avoid exceeding the limit.  
+
+You can't add a route that matches a priority route (a route that's used by your private network).
+
+Routes can be used by only one connection at a time.
+If you enter a non-local/Internet Gateway (IGW) route to a route table, CloudHub 2.0 warns that the route is being used by another connection.
+
+Routes should not overlap with existing routes.
+CloudHub 2.0 warns if you enter a route that overlaps with another route in the route table.
+
+
+[[vpn-limitations]]
+== Anypoint VPN Limitations
+
+Anypoint VPN does not support these features and configurations:
+
+* Network Address Translation (NAT)
+* IPv6
+* IKEv2 with policy-based VPNs
+* A single private space with both Direct Connect and Anypoint VPN connections
+* Advertising a default route (0.0.0.0/0) over BGP or static routing
+
+[[vpn-recommendations]]
+== Anypoint VPN Recommendations
+
+* Adjust the maximum segment size of TCP packets entering the VPN tunnel.
++
+VPN headers require additional space, which reduces the amount of space available for data.
++
+To limit the impact of this behavior, configure your endpoint with TCP MSS Adjustment: 1387 bytes.
+* Reset the `DF` flag on packets.
++
+Packets might carry a Don't Fragment (`DF`) flag, indicating that the packet must not be fragmented. Some VPN devices can override the `DF` flag and fragment packets unconditionally when required. If available, enable the setting `Clear Don't Fragment (DF) Bit`.
+
+[[vpn-ha]]
+== VPN High Availability
+
+Set up a redundant VPN connection to prevent losing connectivity if another 
+VPN or connection device is not available and to allow for maintenance downtime.
+
+The redundant VPN inherits some settings from the initial VPN configuration automatically.
+For example, if the routing type for the initial VPN is dynamic (BGP), the redundant
+VPN is also dynamic.
+
+Other settings, such as the remote ASN for dynamic routing, for the redundant VPN include the values from the initial VPN,
+but you can change the values.
+
+
+To configure a redundant VPN connection, you must:
+
+* Have sufficient entitlements, one for each VPN connection.
++
+See https://help.mulesoft.com/s/article/How-to-Calculate-VPC-and-VPN-License-Requirements[How to Calculate VPC and VPN License Requirements].
```

Comment:
I'm not sure if this is still the case.

Reply:
@vli63127 - it sounds like each private space includes two VPN connection entitlements?

Reply:
the customer should see their separate private space entitlements and VPN entitlements independently, no need to calculate

Reply:
Removed link to KB; changed the hanging paragraph to:
Each private space includes two VPN connection entitlements.
OK? @xuanshi @vli63127